app/scripts/proactive/view/LoginView.js

```diff
@@ -35,10 +35,27 @@ define(
             tryToConnect: function() {
                 if (StudioClient.isLoggedIn()) {
```

Comment:
maybe rename this function to isSessionPresent()

Reply:
actually I did not mean this function

I meant StudioClient.isSessionPresent()